File: medico/Services/Medico.Api/ClientApp/src/app/patientChart/models/visionVitalSigns.ts
export class VisionVitalSigns {
    id: string;
    patientId: string;
    withGlasses: boolean;
    od: number;
    os: number;
    ou: number;
    createDate: any;
    
    constructor() {
        this.withGlasses = false;
        this.createDate = new Date();
    }
}

File: medico/Domain/Medico.Domain/Models/VisionVitalSigns.cs
using System;

namespace Medico.Domain.Models
{
    public class VisionVitalSigns : Entity
    {
        public Guid PatientId { get; set; }

        public Patient Patient { get; set; }

        public bool WithGlasses { get; set; }

        public int Od { get; set; }

        public int Os { get; set; }

        public int Ou { get; set; }

        public DateTime CreateDate { get; set; }
    }
}